### PR TITLE
Removes hardcoding of absolute links with __base_url

### PR DESCRIPTION
**Pull request by lancelafontaine**
All `href`s across the code base were being hardcoded with the absolute link `https://localhost:3000`.

This broke all links for users once the production server was set up.

Solution is to use relative links instead.

### CODE REVIEW

**Comment by Cyberunner23** [comment]
👍

File: views/admin/rooms.ejs [hunk 1]
```diff
@@ -14,11 +14,11 @@
 		<% for(var i=0; i<rooms.length; i++) {%>
 		<tr>
 			<td><%= rooms[i].room %></td>
-		  	<td><a href="<%= __base_url + "/room/" + rooms[i].room %>"><%= rooms[i].title %></a></td>
+		  	<td><a href="/<%= "/room/" + rooms[i].room %>"><%= rooms[i].title %></a></td>
```
Comment (jacobrs) [inline views/admin/rooms.ejs]: we can probably just do `/room/<%= rooms[i].room%>` instead of the extra string append

File: views/room/all.ejs [hunk 2]
```diff
@@ -13,11 +13,11 @@
 			<tr>
 				<td><%= rooms[i].room %></td>
 		  	<td>
-					<a href="<%= __base_url + "/room/" + rooms[i].room %>"><%= rooms[i].title %></a>
+					<a href="/ <%= "/room/" + rooms[i].room %>"><%= rooms[i].title %></a>
```
Comment (jacobrs) [inline views/room/all.ejs]: Same for this could probably be `/room/<% =rooms[i].room %>`

Comment (lancelafontaine) [inline views/room/all.ejs]: Fixed

File: views/admin/rooms.ejs (hunk 1 above)
Comment (lancelafontaine) [inline views/admin/rooms.ejs]: Fixed

**Pull request merged**